playbooks/preflight-check.yml

```diff
@@ -175,3 +175,49 @@
     - name: Duplicate flavors exist
       fail: msg="Duplicated flavor IDs are detected."
       when: flavor_check.stdout != ""
+
+# RLM-1372 Check NFS exsiting
+- name: Check Glance NFS backend on glance
+  hosts: glance_all
+  gather_facts: false
+  user: root
+  tasks:
+    - name: Look for NFS mount on Glance
+      shell: |
+        if mount | grep -i 'nfs'; then
+          mount | grep -i 'nfs'
+        fi
+      register: nfs_mount_found
+      failed_when:
+        - nfs_mount_found.stdout
+
+    - name: Look for NFS config in Glance config
+      shell: |
+        if grep -q 'default_store = file' {{ item }}; then
+          echo 'true'
+        else
+          echo 'false'
+        fi
+      register: nfs_default_store
+      with_items:
+        - "/etc/glance/glance-api.conf"
+      failed_when: nfs_default_store.stdout == 'true'
+
+- name: Check Glance NFS config on infa
+  hosts: "os-infra_hosts"
+  gather_facts: false
+  user: root
+  tasks:
+    - name: Glance NFS config found
+      shell: |
+        if grep -q 'glance_default_store: nfs' {{ item }}; then
```

Comment:
Why are you doing a shell script instead of evaluating just glance_default_store ?

Reply:
Also nfs mounts in infra would likely be configured as  glance_default_store: file so that logic seems wrong as glance_default_store: nfs is actually a invalid configuration